apps/sync/js/fxsync-webcrypto/fxsyncwebcrypto.js

```diff
@@ -0,0 +1,279 @@
+'use strict';
+
+/* global StringConversion, KeyDerivation, crypto */
+/* exported FxSyncWebCrypto */
+
+// WebCrypto-based client for Firefox Sync.
```

Comment:
OK, I merged them into one readme in the file, and put it in the folder. Unfortunately, markdown hyperlinks to long URLs in 80-char width will probably not work in a markdown viewer, but it's clear what the URLs should be when you open the text file. (prepared this locally, will push later today)